pkgs/top-level/release.nix

```diff
@@ -11,7 +11,7 @@
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; revision = "0000000000000000000000000000000000000000"; }
 , officialRelease ? false
   # The platforms for which we build Nixpkgs.
-, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ]
```

Comment:
Nit: While technically not much of a difference (right now) this should probably be in the `limitedSupportedSystems` set. We don't provide "basic" expressions such as GHC just yet on `aarch64-darwin` which really can't be called "supported" in that case. I think this also reflects the current status much better also in regards to the Tiers RFC (which isn't required for adding it here as far as I am concerned).

Reply:
limitedSupportedSystems is a misnomer, as it won't build any packages if we put it only there.

limitedSupportedSystems will only build bootstrap tooling: https://hydra.nixos.org/eval/1673896?filter=.i686-linux&compare=1673856&full=#tabs-still-succeed